test/test.filters.js

```diff
@@ -640,6 +640,10 @@ describe('Twig.js Filters ->', function () {
             const testTemplate = twig({data: '{{ \'12345\'|slice(1, 2) }}'});
             testTemplate.render().should.equal('23');
         });
```

Comment:
Please add more test cases to cover both the optional first and second parameters.